### ISSUE TITLE
ImageControl displays 0 if 0 is the value passed

### ISSUE THREAD

**Issue by tomjn** [humanmade/block-editor-components#17]
If an invalid numeric value such as `0` is passed as the value prop, then `ImageControl` will print it.

Tested on WP 5.9

**Comment by tomjn** [comment]
I've gotten around this using:

```jsx
			<PanelRow>
				<ImageControl
					value={ backgroundImgID !== 0 ? null : backgroundImgID }
					onChange={ ( image ) => editPost( { meta: { [ META_KEY_AWARDS_BG_IMG ]: image?.id } } )  }3
				/>
			</PanelRow>
```

Note I also tested with a non-numeric value `"banana"` and it tried to query the media endpoint for it and got an error response. It didn't print the text banana but it did give me the option to set a new image and to remove it

**Comment by tfrommen** [comment]
Fixed in #21.
